patches/server/0005-Paper-config-files.patch

```diff
@@ -1899,6 +1899,8 @@ index 0000000000000000000000000000000000000000..385ca2c1022e0985550a5cc2bbff953f
 +        public int dryFarmland = 1;
 +        public Table<EntityType<?>, String, Integer> sensor = Util.make(HashBasedTable.create(), table -> table.put(EntityType.VILLAGER, "secondarypoisensor", 40));
 +        public Table<EntityType<?>, String, Integer> behavior = Util.make(HashBasedTable.create(), table -> table.put(EntityType.VILLAGER, "validatenearbypoi", -1));
++        public int maxFluidTicks = 65536;
```

Comment:
I don't think that putting this in tick-rates is a really good idea, as everything else in here is a "how often per tick", not a "how many per tick" deal

Reply:
Yeah I thought the same thing. But I don't think any other category fits either. Any recommendations?